topology/University of Nebraska/Nebraska-Lincoln/UNLCachingInfrastructure.yaml

```
@@ -169,3 +169,32 @@ Resources:
         Description: HCC StashCache Origin Server
     AllowedVOs:
       - HCC
+      
+  LIGO-StashCache-Origin:
+    Active: true
+    Description: LIGO StashCache origin
+    ID: 
```

Oops, one last thing!

```suggestion
    ID: 1021
```